src/rebar_prv_dialyzer.erl

```diff
@@ -409,13 +420,29 @@ base_plt_files(State) ->
 base_plt_apps(State) ->
     get_config(State, base_plt_apps, [erts, crypto, kernel, stdlib]).
 
-update_base_plt(State, BasePlt, Output, BaseFiles) ->
-    case read_plt(State, BasePlt) of
-        {ok, OldBaseFiles} ->
-            check_plt(State, BasePlt, Output, OldBaseFiles, BaseFiles);
-        error ->
-            _ = filelib:ensure_dir(BasePlt),
-            build_plt(State, BasePlt, Output, BaseFiles)
+update_base_plt(State, BasePlt, Output) ->
+    case is_update_base_plt_active(State) of
+        false ->
+            {ok, BaseFiles} = plt_files(BasePlt),
```

Comment:
We should handle the situation where the base PLT does not exist. We may want to test that depending what action is taken. I would prefer to build project PLT from scratch in this situation.